### ISSUE TITLE
Browser doesn't recognize edits unless additional edits from the browser are made

### ISSUE THREAD

**Issue by GOKOP** [jlebon/textern#60]
After editing the field through Textern the edited text does appear in the field, but the webpage behaves as if it was not there.
For example, on Github I can't submit a comment written entirely through Textern until I change something afterwards; if I write something beforehand and then edit through Textern, without further manipulations only stuff that was written before gets submitted.
This also makes Textern unusable on Reddit because it resets the filed to whatever it was before using Textern when I click it.

**Comment by jlebon** [comment]
Yeah, definitely noticed this as well in some websites (oddly not on GitHub though). Might just need to kick event listeners after each update. Will look into this.

**Comment by cassava** [comment]
I also noticed this in the SoGO client from [Mailcow](https://mailcow.github.io/mailcow-dockerized-docs/). When writing an email it probably saves the draft after every few keystrokes, and as soon as I used TexTern it stays at the last state. Took me a while to figure out what was happening.

**Comment by MaxGyver83** [comment]
Same problem in [TiddyWiki](https://tiddlywiki.com/). When I edit a "tiddler" with textern and have the preview pane activated, I don't see any changes made in textern in the preview until I make some changes in the browser textedit area afterwards.

**Comment by MaxGyver83** [comment]
This is also a problem in Microsoft Teams. The message field contains some placeholder text:
![image](https://user-images.githubusercontent.com/13970628/174659924-f8c8d506-e1d8-4a88-8d4f-35ca476bea76.png)

When I enter some text using textern, the placeholder text won't be cleared and the entered text is hard to read:
![image](https://user-images.githubusercontent.com/13970628/174660034-092968bc-c598-424a-b325-f8558fc54405.png)

I need to enter more text afterwards to get the placeholder text removed.

**Comment by MaxGyver83** [comment]
> Might just need to kick event listeners after each update. Will look into this.

@jlebon : Is this project still alive? Could you look into this if you find some time?

**Comment by MaxGyver83** [comment]
@jlebon : Finally, I could find a really easy way to fix this on my own. See pull request #87.
